mockgen/mockgen.go

```diff
@@ -115,6 +116,14 @@ func main() {
 		return
 	}
 
+	if len(*interfaces) > 0 {
```

Comment:
maybe this should go in `sourceMode` like `p.excludeNamesSet = parseExcludeInterfaces(*excludeInterfaces)`?